app/v2/notifications/post_notifications.py

```diff
@@ -222,3 +282,27 @@ def get_reply_to_text(notification_type, form, template):
         reply_to = template.get_reply_to_text()
 
     return reply_to
+
+
+def get_precompiled_letter_template(service_id):
+    template = Template.query.filter_by(
+        service_id=service_id,
+        template_type=LETTER_TYPE,
+        hidden=True
+    ).first()
```

Comment:
One? We should only ever have one and if we don't something is wrong?

BUT it shouldn't break the system